mason.sh

```diff
@@ -67,6 +71,9 @@ elif [ ${MASON_PLATFORM} = 'ios' ]; then
     export MASON_IOS_CFLAGS="${MIN_SDK_VERSION_FLAG} -isysroot ${MASON_SDK_PATH} -arch armv7 -arch armv7s -arch arm64"
     if [[ ${MASON_PLATFORM_VERSION%%.*} -ge 9 ]]; then
         export MASON_IOS_CFLAGS="${MASON_IOS_CFLAGS} -fembed-bitcode"
+        export MASON_DYNLIB_SUFFIX="tbd"
```

Comment:
This change breaks building for iOS 9 on OS X 10.10 and below. If building for iOS 9 on OS X 10.11 is broken, you should add a conditional that looks at the OS X version.